packages/frontend/component/src/components/not-found-page/not-found-page.tsx

```diff
@@ -14,46 +16,63 @@ import {
 
 export interface NotFoundPageProps {
   user?: User | null;
+  signInComponent?: JSX.Element;
   onBack: () => void;
   onSignOut: () => void;
 }
 export const NotFoundPage = ({
   user,
   onBack,
   onSignOut,
+  signInComponent,
 }: NotFoundPageProps) => {
   const t = useAFFiNEI18N();
+  const [isSmallScreen, setIsSmallScreen] = useState(false);
+
+  useEffect(() => {
+    const checkScreenSize = () => {
+      setIsSmallScreen(window.innerWidth <= 1024);
+    };
+    checkScreenSize();
+    window.addEventListener('resize', checkScreenSize);
+    return () => window.removeEventListener('resize', checkScreenSize);
+  }, []);
```

Comment:
Is it possible to use media query instead?

Reply:
Of course it is possible, but `<AffineOtherPageLayout/>` needs to be refactored
[AffineOtherPageLayout Link](https://github.com/toeverything/AFFiNE/blob/97c4ae48b5e7736dffa452e22cc52ba119a77db0/packages/frontend/component/src/components/affine-other-page-layout/layout.tsx#L11)